docs/vm-images.md

```diff
@@ -1,20 +1,60 @@
 # Supported and Tested VM Images
 
+* [General Information](#general-information)
 * [HPC CentOS 7](#hpc-centos-7)
 * [HPC Rocky Linux 8](#hpc-rocky-linux-8)
 * [Debian 11](#debian-11)
 * [Ubuntu 20.04 LTS](#ubuntu-2004-lts)
 * [Windows](#windows)
 * [Other Images](#other-images)
+* [Slurm on GCP](#slurm-on-gcp)
 
 For information on customizing VM images with extra software and configuration
 settings, see [Building Images](image-building.md).
 
 Please see the [blueprint catalog](https://cloud.google.com/hpc-toolkit/docs/setup/hpc-blueprint-catalog) for examples.
 
-For Slurm images, please see [SchedMD's GitHub repository](https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md#public-image).
+## General Information
 
-## HPC CentOS 7
+### Instance Image
+
+> **_Note:_** This information is true for most source modules, but there are some sources that have their own image specification.  Please read the documentation for any source module utilized.
```

Comment:
I don't think we should include this change in this PR because it's really unrelated. But I will defer to @cboneti who plans a follow-up review for this PR.